HappyFox Test Automation Task Java /HelpDeskProductTests/src/main/java/CommonTests/CommonTests1.java

```diff
@@ -0,0 +1,208 @@
+package CommonTests;
+
+import java.io.File;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.PageFactory;
+import org.testng.Assert;
+import org.testng.annotations.BeforeSuite;
+import org.testng.annotations.Test;
+
+import Admin_Portal.AdminPortalTest1;
+import Admin_Portal.AdminPortalTest1stPage;
+import Admin_Portal.AdminPortalTest2ndPage;
+import Admin_Portal.Login_Page;
+import CustomerPortal.SupportPortalPage;
+import webdriverbase.BaseTest;
+
+public class CommonTests1 extends BaseTest{
+	
+public Login_Page hploginpage;
+public SupportPortalPage hpSupportPortalPage;
+public AdminPortalTest1stPage hpAdminPortalTest1;
+public AdminPortalTest2ndPage hpAdminPortalTest2;
+	
+public String AdminPortalURL = "https://interview.supporthive.com/staff";
+public static String SupportTicketURL = "https://interview.supporthive.com/new/";
+public static String browser = "Chrome";
+public static String username = "interview_agent";
+public static String password = "Interview@123";
+public static String statusName = "Issue created";
+public static String priorityName = "Assistance required";
+public static String Subject = "Test Ticket raised by XYZ";
+public static String Message = "Hi, i am having certain issues in the Happy fox portal. Can you please help me.";
+public static String FullName = "Reese Harrold";
+public static String Email = "testno100@gmail.com";
+
+public void loginasAdmin() {
+    
+    String Methodname = new Object(){}.getClass().getEnclosingMethod().getName(); 
+	System.out.println("Start of LoginAsAdmin "+"MethodName : "+Methodname); 
+
+	try
+	{	
+		hploginpage = new Login_Page(getDriver());
+		hploginpage.navigateToHappyFoxHomePageURL(AdminPortalURL);
+		hploginpage.enterUsername(username);
+		System.out.println("username entered");
+		hploginpage.enterPassword(password);
+		System.out.println("password entered");
+		hploginpage.clickLoginbutton();
+		System.out.println("login clicked");
+		hpAdminPortalTest1=hploginpage.validatePendingTicketsTitle();
+		
+		
+	}
+ 
+	catch(Throwable t)
+	{
+		t.printStackTrace();
+		Assert.fail("Error in " + Methodname + " : " + t.getMessage());
+	}
+	
+	
+}
+  
+  public void CreatingSupportTicket() {
+	  
+	  String Methodname = new Object(){}.getClass().getEnclosingMethod().getName(); 
+	  System.out.println("Start of Client support ticket "+"MethodName : "+Methodname); 
+	  
+	  try 
+	  {
+		    
+		  hpSupportPortalPage = new SupportPortalPage(getDriver());
+		  hpSupportPortalPage.navigateToHappyFoxSupportPortalURL(SupportTicketURL);
+		  hpSupportPortalPage.enterSubject(Subject);
+		  hpSupportPortalPage.enterMessage(Message);
+		  hpSupportPortalPage.clickAddCC();
+		  hpSupportPortalPage.clickAddBCC(); 
+		  hpSupportPortalPage.enterFullName(FullName);
+		  hpSupportPortalPage.enterEmail(Email);
+		  hpAdminPortalTest2=hpSupportPortalPage.clickCreateTicket();
+		  System.out.println("Ticket created");
```

Comment:
printing logger is good, but how can we make sure ticket is created and assertion is missing